code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -27,22 +27,62 @@
 
 	if(iscarbon(L))
 		var/mob/living/carbon/C = L
-		if(C.get_blood_id() == "blood" && (method == INJECT || (method == INGEST && C.dna && C.dna.species && (DRINKSBLOOD in C.dna.species.species_traits))))
+		var/blood_id = C.get_blood_id()
+		if((blood_id == "blood" || blood_id == "jellyblood") && (method == INJECT || (method == INGEST && C.dna && C.dna.species && (DRINKSBLOOD in C.dna.species.species_traits))))
 			if(!data || !(data["blood_type"] in get_safe_blood(C.dna.blood_type)))
-				C.reagents.add_reagent("toxin", reac_volume * 0.5)
+				C.reagents.add_reagent("bonehurtingjuice", reac_volume * 0.5)
```

Comment:
Okay but seriously, bonehurtingjuice does nothing but deal staminaloss and make the person spam "oof", "ouch", and "owie". Dealing toxinloss directly would be very much preferable to this.

Reply:
I would prefer a different chem than free healing for slimepeople